packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -919,8 +940,19 @@ export const DEFAULT_TOKEN_ADJACENCY_GRAPH_BY_CHAIN_ID = valueByChainId<TokenAdj
                 builder.addBidirectional(MAINNET_TOKENS.OHMV2, MAINNET_TOKENS.BTRFLY);
                 // Lido
                 builder.addBidirectional(MAINNET_TOKENS.stETH, MAINNET_TOKENS.wstETH);
+                // Synthetix Atomic Swap
+                builder.addCompleteSubgraph([
```

Comment:
Interested to see how the `gasUsed` is for this (you can console.log it in [Sampler Metrics](https://github.com/0xProject/0x-api/blob/master/src/utils/sampler_metrics.ts#L27))

Reply:
Mainnet:
- sUSD -> WETH (sell) : 162m
- sUSD -> WETH (buy) : 165m
- WETH -> sUSD (sell): 168m
- WETH -> sUSD (buy): 158m

Optimism
- sUSD -> WETH (sell) : 107m
- sUSD -> WETH (buy) : 153m
- WETH -> sUSD (sell) : 338m
- WETH -> sUSD (buy) : 288m